nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -212,7 +212,20 @@ environment.systemPackages = [
        Be aware that backwards state migrations are not supported by Deluge.
      </para>
    </listitem>
-
+   <listitem>
+     <para>
+       Now nginx web server started in sandbox mode. By default, recording is
+       allowed in <literal>/var/spool/nginx</literal>. To allow writing to other folders,
+       use <literal>systemd.services.nginx.serviceConfig.ReadWritePaths</literal>
+       <programlisting>
+systemd.services.nginx.serviceConfig.ReadWritePaths = [ "/var/www" ];
+       </programlisting>
+       Breaks the nginx lua and pagespeed modules. To correct work need disable <literal>MemoryDenyWriteExecute</literal>
```

Comment:
Can we test if these modules are enabled, and only enable `MemoryDenyWriteExecute` when they're not enabled?

Reply:
I don't know how to implement

Reply:
@flokli simple worked variant 
```
  nixpkgs.overlays = [
    (self: super: {
      php = pkgs.php72;
      phpPackages = pkgs.php72Packages;
      nginx = super.nginx.override {
        withDebug = false;
        withStream = false;
        withMail = false;
        modules = [
          pkgs.nginxModules.lua
        ];
      };
    })
  ];
```
and
```
MemoryDenyWriteExecute = (if pkgs.nginx.modules == [ pkgs.nginxModules.lua ] then false else true);
```
If change to 
```
        modules = [
          pkgs.nginxModules.lua
          pkgs.nginxModules.rtmp
        ];
```
Not worked.
What other method is there?

Reply:
Added test to check are enabled modules.